src/server.cpp

```diff
@@ -383,7 +392,7 @@ Response InternalServer::build_500(const std::string& msg)
 {
   kainjow::mustache::data data;
   data.set("error", msg);
-  Response response(m_root, true, false, false);
+  Response response(m_root, true, false, false, m_blockExternalLinks);
```

Comment:
I would not block external here. 500 means internal server error.
If we have this, we probably don't want to insert so js that redirect to us.

(I'm not sure we have links in `_500_html`, but be future proof, we may want to add a link to a bugtracker or something else)

Reply:
Right.